pkg/encyclopedia/aircraft.go

```diff
@@ -1001,7 +1000,7 @@ var aircraftData = []Aircraft{
 		PlatformDesignation: "MiG-31",
 		TypeDesignation:     "MiG-31",
 		NATOReportingName:   "Foxhound",
-		threatRadius:        80 * unit.NauticalMile,
+		threatRadius:        highThreatRadius,
```

Comment:
This should also be `mediumThreatRadius`.

The only `highThreatRadius` should be either Bluefor (AMRAAM/Phoenix) or the PL-12 (JF-17 only I think)